### PR TITLE
fix(README): grammar

### PR DESCRIPTION
**Pull request by gargarchit**
Signed-off-by: Archit garg4.ag@gmail.com

I found some Improper word choice as well as Improper formatting for that I have made changes and corrected that so, now It will be easier for everyone to read.
Please Merge this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Invokes _lerna_ to run the test suite.
   </a>
 </p>
 
-Accord Project is an open source, non-profit, initiative working to transform contract management and contract automation by digitizing contracts. Accord Project operates under the umbrella of the [Linux Foundation][linuxfound]. The technical charter for the Accord Project can be found [here][charter].
+Accord Project is an open-source, non-profit, initiative working to transform contract management and contract automation by digitizing contracts. Accord Project operates under the umbrella of the [Linux Foundation][linuxfound]. The technical charter for the Accord Project can be found [here][charter].
```
Comment (irmerk) [inline README.md]: This can be removed.

File: README.md [hunk 2]
```diff
@@ -324,7 +324,7 @@ Accord Project is an open source, non-profit, initiative working to transform co
 
 ## Contributing
 
-The Accord Project technology is being developed as open source. All the software packages are being actively maintained on GitHub and we encourage organizations and individuals to contribute requirements, documentation, issues, new templates, and code.
+The Accord Project technology is being developed as an open-source. All the software packages are being actively maintained on GitHub and we encourage organizations and individuals to contribute requirements, documentation, issues, new templates, and code.
```
Comment (irmerk) [inline README.md]: This can be removed.

Comment (gargarchit) [reply README.md]: Updated Changes

File: README.md (hunk 1 above)
Comment (gargarchit) [reply README.md]: Updated Changes

**Comment by gargarchit** [comment]
@irmerk Thanks for review I updated changes and added DCO

File: README.md [hunk 3]
```diff
@@ -324,7 +324,7 @@ Accord Project is an open source, non-profit, initiative working to transform co
 
 ## Contributing
 
-The Accord Project technology is being developed as open source. All the software packages are being actively maintained on GitHub and we encourage organizations and individuals to contribute requirements, documentation, issues, new templates, and code.
+The Accord Project technology is being developed as an open source. All the software packages are being actively maintained on GitHub and we encourage organizations and individuals to contribute requirements, documentation, issues, new templates, and code.
```
Comment (irmerk) [inline README.md]: This can be removed.

**Comment by irmerk** [comment]
One change, then this can be merged.

File: README.md (hunk 3 above)
Comment (jeromesimeon) [reply README.md]: `as an open source` does not seem correct.

Comment (gargarchit) [reply README.md]: Yeah I have updated this

**Comment by irmerk** [comment]
Needs DCO again

**Comment by gargarchit** [comment]
@irmerk Done

**Comment by gargarchit** [comment]
@irmerk I have Updated all changes

**Comment by gargarchit** [comment]
Thank You @irmerk

**Comment by gargarchit** [comment]
Thank You @irmerk

**Pull request merged**